programs/rodeos/ship_client.hpp

```diff
@@ -33,6 +33,7 @@ struct connection_callbacks {
    virtual bool received(ship::get_blocks_result_v1& result, eosio::input_stream bin) { return true; }
    virtual bool received(ship::get_blocks_result_v2& result, eosio::input_stream bin) { return true; }
    virtual void closed(bool retry) = 0;
+   virtual void quited() = 0;
```

Comment:
Remove `quited`. `closed` already has a path that calls `appbase::app().quit()`. Add another option to `closed` on if to call `quit`.

Reply:
Initially I added a `quitting` parameter to `closed`. Then I was worry that made `closed` do two things and a specific `quitted`  might be cleaner. Changed to use an additional option to `closed`.